packages/ckeditor5-table/src/tablecellproperties/tablecellpropertiesediting.ts

```diff
@@ -76,7 +76,11 @@ export default class TableCellPropertiesEditing extends Plugin {
 		const schema = editor.model.schema;
 		const conversion = editor.conversion;
 
-		editor.config.define( 'table.tableCellProperties.defaultProperties', {} );
+		editor.config.define( 'table.tableCellProperties.defaultProperties', {
```

Comment:
These default values should be implemented in https://github.com/ckeditor/ckeditor5/blob/2b9341bb7932aaf3c1e55132abe940130d569cf9/packages/ckeditor5-table/src/utils/table-properties.ts#L100-L108

Reply:
Yep, I was already doing this because I reopened old PR.